src/lib/y2storage/dialogs/guided_setup.rb

```diff
@@ -20,104 +20,77 @@
 # find current contact information at www.novell.com.
 
 require "yast"
-require "y2storage"
-require "ui/installation_dialog"
+require "y2storage/disk_analyzer"
+require "y2storage/refinements/size_casts"
+require "y2storage/dialogs/guided_setup/select_disks"
+require "y2storage/dialogs/guided_setup/select_root_disk"
+require "y2storage/dialogs/guided_setup/select_scheme"
+require "y2storage/dialogs/guided_setup/select_filesystem"
 
 module Y2Storage
   module Dialogs
+    # Class to control the guided setup workflow.
+    #
     # Calculates the proposal settings to be used in the next proposal attempt.
-    class GuidedSetup < ::UI::InstallationDialog
+    class GuidedSetup
+      Yast.import "Sequencer"
+
+      using Y2Storage::Refinements::SizeCasts
       # @return [ProposalSettings] settings specified by the user
       attr_reader :settings
+      # Currently probed devicegraph
```

Comment:
An error. I have changed that. Now `GuidedSetup` is called with `storage_manager.probed`.